hw/xfree86/common/xf86platformBus.h

```diff
@@ -122,14 +103,6 @@ _xf86_get_platform_device_int_attrib(struct xf86_platform_device *device, int at
 extern _X_EXPORT Bool
 xf86PlatformDeviceCheckBusID(struct xf86_platform_device *device, const char *busid);
 
-extern void xf86platformVTProbe(void);
-extern void xf86platformPrimary(void);
```

Comment:
Are we sure that no one outside is using it?

Reply:
No. They're not _X_EXPORT'ed.